### ISSUE TITLE
Distorting the image on new images to predict

### ISSUE THREAD

**Issue by handav** [Microsoft/FERPlus#12]
@ebarsoumMS @cha-zhang Is distorting the images a necessary step for new images I'm predicting on? Any other guidance for preprocessing new images to feed into the trained model?

**Comment by amirhfarzaneh** [comment]
If you mean data augmentation by distortion then no! Data augmentation is a common practice only for the training set. Some people also do data augmentation on the test set to see if their network generalizing well or not. On the other hand, pre-processing is different than data augmentation. The authors apply pre-processing to enhance the contrast between the face and background. They are actually doing histogram equalization. You want your test set and train set to be of same distribution and since pre-processing is already applied on training set you have to apply it on the test set as well. You might notice performance decrease if you don't pre-process your test set. You can read more about other pre-processing techniques for FER applications in this survey: https://arxiv.org/abs/1804.08348

**Comment by handav** [comment]
No, apologies for being unclear - I mean what preprocessing should I be doing on new images that I’m using the trained model to predict? Cropping to the face and resizing to 64x64 is clear, and I’m also calling the normalize and preprocessing function on it from img_util. Does that sound right?

**Comment by amirhfarzaneh** [comment]
There is no cropping to the face that I'm aware of!
- [x] resize to 64x64
- [x] normalize
- [x] pre-process

These three steps are necessary!
